### PR TITLE
SEO-2750 - ASP.NET Core Grid images alt text

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
|Title|Description|
|:--|:--|
|**Task Category** | Need to rename 18 image alt text|
|**Content task link/ Mail screenshot** | N/A |
|**Hotfix PR link** | https://github.com/syncfusion-content/asp.net-core-docs/pull/232
|**Ticket link/Task link** |https://syncfusion.atlassian.net/browse/SEO-2750 |
|**Excel/SharePoint link** | N/A |

### CODE REVIEW

File: aspnet-core/Grid/Columns.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-description: columns
+title: Columns in ASP.NET Core Grid Control | Syncfusion
+description: Learn here all about Column support in Syncfusion Essential ASP.NET Core Grid control, its elements, and more.
 platform: aspnet-core
 control: Grid
 documentation: ug
 ---
 
-# Columns
+# Columns in Essential ASP.NET Core Grid
```
Comment (Aishwarya-Ganesan) [inline aspnet-core/Grid/Columns.md:10]: Columns in ASP.NET Core Grid

Comment (Sharly-Atieno) [reply aspnet-core/Grid/Columns.md]: essential removed

**Comment by Sharly-Atieno** [comment]
rebuild

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are Fine.

**Pull request merged**